pkg/operation/botanist/addons.go

```diff
@@ -431,6 +431,16 @@ func (b *Botanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart, erro
 		return nil, err
 	}
 
+	apiserverProxyConfig := map[string]interface{}{
```

Comment:
Probably, this new component should be added to the required system components (https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/health_check.go#L320) for the care controller health check.

However, the health checks for the system components are improved as part of #2422 (which will probably be merged before this PR), so that only the conditions of the `ManagedResource`s are checked. This means that you don't have to adapt the PR here now if we merge it after #2422. Just wanted to bring it to your attention.